lib/members.js

```diff
@@ -399,8 +435,99 @@ Membership.prototype._emitUpdated = function _emitUpdated(updates) {
     this.emit('updated', updates);
 };
 
+function MembershipUpdateTracker(options) {
```

Comment:
this class just tracks stuff in memory. I presume it will be used to read from in some future PR.